fxcore/chain.json

```diff
@@ -0,0 +1,213 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "fxcore",
+  "chain_id": "fxcore",
+  "website": "https://functionx.io",
+  "pretty_name": "f(x)Core",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "fx",
+  "daemon_name": "fxcored",
+  "node_home": "$HOME/.fxcore",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "FX",
+        "fixed_min_gas_price": 4200000000000,
+        "low_gas_price": 400000000000,
+        "average_gas_price": 4200000000000,
+        "high_gas_price": 500000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "FX"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/functionx/fx-core",
+    "recommended_version": "v4.2.1",
+    "compatible_versions": [
+      "v4.2.1"
+    ],
+    "binaries": {
+      "linux/amd64": "https://github.com/functionx/fx-core/releases/download/v4.2.1/fx-core_Linux_x86_64.tar.gz",
+      "linux/arm64": "https://github.com/functionx/fx-core/releases/download/v4.2.1/fx-core_Linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/functionx/fx-core/releases/download/v4.2.1/fx-core_Darwin_x86_64.tar.gz",
+      "darwin/arm64": "https://github.com/FunctionX/fx-core/releases/download/v4.2.1/fx-core_Darwin_arm64.tar.gz",
+      "windows/amd64": "https://github.com/functionx/fx-core/releases/download/v4.2.1/fx-core_Windows_x86_64.zip"
+    },
+    "genesis": {
+      "name": "fxv1",
+      "genesis_url": "https://github.com/functionx/fx-core/raw/main/public/mainnet/genesis.json"
+    },
+    "cosmos_sdk_version": "0.46",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "ibc_go_version": "6.1.0",
+    "versions": [
```

Comment:
thanks for adding the versions array. Last request is to correct the order from first version at the top, and latest/current version at the bottom.